ksqldb-common/src/main/java/io/confluent/ksql/util/QueryApplicationId.java

```diff
@@ -30,7 +30,7 @@ public static String buildSharedRuntimeId(
       final boolean persistent,
       final int sharedRuntimeIndex
   ) {
-    final String queryAppId = buildPrefix(config, persistent) + "-" + sharedRuntimeIndex;
```

Comment:
seriously great catch -- would have sucked to get stuck with this awkward looking thing